docs/guides/create-library.md

```diff
@@ -33,8 +49,7 @@ shortcuts if you don't want to focus on technical setup.
 ## 1. Project setup
 
 Let's dive right in. Start a terminal and change to the directory
-you want to work in. For this tutorial we'll use `~/alva/` in examples,
-screenshots and screencasts but any location on your computer works.
+you want to work in. For this tutorial we'll use `~/alva-guides/` but any location works.
```

Comment:
if we change that directory -> change also:
2d) ~/alva/component-library/package.json -> ~/alva-guides/component-library/package.json